### PR TITLE
Feature/styles refactor

### PR DESCRIPTION
**Pull request by sorodrigo**
This PR does the following:
- Repositions the scroll down animation.
- Repositions the introduction heading using a new class `c-intro-heading`.
- Refactors the `c-content` from the section layout (`l-layout`).

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -53,27 +53,27 @@
...
+            <div class="heading-item">disappearing</div>
+            <div class="heading-item">world</div>
           </h1>
         </div>
       </div>
     </div>
 
     <div class="l-section -parallax">
-      <div class="section-content row column">
+      <div class="c-content row column">
```
Comment (mbarrenechea) [inline src/index.html:74]: Row and column at the same time? A little bit strange

File: src/index.html [hunk 2]
```diff
@@ -53,27 +53,27 @@
...
-      <div>scroll down to find out more</div>
+      <div class="footnote">scroll down to find out more</div>
     </div>
 
     <!-- Canvas world with layers -->
     <div id="world" class="l-section -fixed"></div>
 
     <div class="l-section -parallax">
-      <div class="section-content row column">
+      <div class="row column">
```
Comment (mbarrenechea) [inline src/index.html:63]: Row and column at the same time? A little bit strange

File: src/index.html [hunk 3]
```diff
@@ -107,23 +107,23 @@
...
+            </article>
         </div>
       </div>
     </div>
 
     <div id="deerPhoto" class="l-section -parallax"></div>
 
     <div id="worldVis" class="l-section -parallax">
-      <div class="section-content row column">
+      <div class="c-content row column">
```
Comment (mbarrenechea) [inline src/index.html:124]: It's everywhere....ajjaja

**Review by mbarrenechea — CHANGES_REQUESTED** [review]

**Comment by sorodrigo** [comment]
I removed a redundant `column small-12` inside of a `row column`. Now it should comply with foundation docs.

**Review by mbarrenechea — APPROVED** [review]
As long as it's on the documentation, let's merge it

**Pull request merged**